contracts/WithdrawManager.sol

```diff
@@ -91,7 +93,10 @@ contract WithdrawManager {
         (checkSuccess, callSuccess) = BLS.verifySingle(
             signature,
             pubkey,
-            BLS.hashToPoint(appID, abi.encodePacked(msg.sender))
+            BLS.hashToPoint(
+                domain.domainSeparator(),
```

Comment:
Because the `chainid` can change (https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.0/contracts/drafts/EIP712.sol#L26-L27), we will need to check the Rollup contract every time this is run.

Reply:
Added a note for this to `IEIP712.domainSeparator()` docstring.